packages/docs-site/src/library/pages/components/masthead.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Masthead
+description: A top level section displayed at the top of the page
+header: true
+---
+
+import { Icons, Masthead, Tab, TabSet } from '@royalnavy/react-component-library'
+import DataTable from '../../../components/presenters/data-table'
+import CodeHighlighter from '../../../components/presenters/code-highlighter'
+
+
+# Overview
+The Masthead is a simple section at the top of the page that allows the developer to communicate the service logo, name and offer a search bar.
+
+## Usage
+
+<TabSet>
+
+<Tab title="Design">
+
+  ## Design
+  TBD.
```

Comment:
We need to make sure design docs are added before merge to develop or it'll block releases.

Reply:
Yeah, and thats going to be a thing going forward for all components that we need to discuss as a team